tap/tlstapper/tls_tapper.go

```diff
@@ -0,0 +1,174 @@
+package tlstapper
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/cilium/ebpf/perf"
+	"github.com/cilium/ebpf/rlimit"
+	"github.com/go-errors/errors"
+	"github.com/up9inc/mizu/shared/logger"
+)
+
+//go:generate go run github.com/cilium/ebpf/cmd/bpf2go tlsTapper bpf/tls_tapper.c -- -O2 -g -D__TARGET_ARCH_x86
```

Comment:
What about arm64? Because we recently added official support.

Reply:
For now, only x86

Reply:
OK x86 specific for now.